### PR TITLE
Extending sample file for supporting resiliency module

### PR DESCRIPTION
**Pull request by sakshi-garg1**
# Description
Extending sample file for supporting resiliency module

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
|https://github.com/dell/csm/issues/739 |

# Checklist:

- [x] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [ ] I have verified that new and existing unit tests pass locally with my changes
- [ ] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 90% code coverage
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have maintained backward compatibility

### CODE REVIEW

File: operatorconfig/driverconfig/powerflex/v2.7.0/controller.yaml [hunk 1]
```diff
@@ -52,6 +52,9 @@ rules:
   - apiGroups: [""]
     resources: ["secrets"]
     verbs: ["get", "list"]
+  - apiGroups: ["volumegroup.storage.dell.com"]
```
Comment (adarsh-dell) [inline operatorconfig/driverconfig/powerflex/v2.7.0/controller.yaml:55]: @sakshi-garg1 Is there anything to do with this sidecar for resiliency module?

Comment (sakshi-garg1) [reply operatorconfig/driverconfig/powerflex/v2.7.0/controller.yaml]: removed this code

**Review by adarsh-dell — APPROVED** [review]

**Comment by adarsh-dell** [comment]
@suryagupta4 Although this one looks good to me but can you please cross check once more, the args with helm before you approve it.

**Comment by suryagupta4** [comment]
> @suryagupta4 Although this one looks good to me but can you please cross check once more, the args with helm before you approve it.

Thanks Adarsh. Found indentation issue at L324.

**Review by suryagupta4 — APPROVED** [review]
lgtm

**Pull request merged**